src/WaitCommand.php

```diff
@@ -42,15 +42,28 @@ public function wait($files, Env $env, PlatformShFacade $facade, SymfonyStyle $i
             throw new \RuntimeException(self::CIRCLE_PULL_REQUEST_ERROR);
         }
 
-        $url = $facade->waitFor($platformId, 'pr-' . $prNum, $sha);
-        $url = rtrim($url, '/');
-        foreach ($files as $file) {
+        $environment = $facade->waitFor($platformId, 'pr-' . $prNum, $sha);
```

Comment:
This defeats the idea of the facade encapsulating all Platform.sh related logic.

But I don't see no reason why `waitFor` couldn't return an array of urls.

Reply:
I see your point.

If we return an array of urls do you have any suggestions regarding how to distinguish between the public url and the route urls?

Reply:
Let the public be index 0. Also fixes the indexing of route urls.

I notice I forgot to document the `waitFor` function, else I would have added a comment about documenting the array...